crates/quickjs-wasm-rs/src/js_binding/value.rs

```diff
@@ -147,13 +149,60 @@ impl Value {
     }
 
     pub fn as_str(&self) -> Result<&str> {
+        let buffer = self.as_wtf8_str_buffer();
+        str::from_utf8(buffer).map_err(Into::into)
+    }
+
+    pub fn as_str_lossy(&self) -> std::borrow::Cow<str> {
+        let mut buffer = self.as_wtf8_str_buffer();
+        match str::from_utf8(buffer) {
+            Ok(valid) => Cow::Borrowed(valid),
+            Err(mut error) => {
+                let mut res = String::new();
+                loop {
+                    let (valid, after_valid) = buffer.split_at(error.valid_up_to());
+                    res.push_str(unsafe { str::from_utf8_unchecked(valid) });
+                    const REPLACEMENT_CHAR: char = '\u{FFFD}';
```

Comment:
I believe this can be imported from https://doc.rust-lang.org/stable/std/primitive.char.html#associatedconstant.REPLACEMENT_CHARACTER